packages/lab/src/tabs-next/TabNext.css

```diff
@@ -0,0 +1,120 @@
+/* Class applied to root Tab element */
+.saltTabNext {
+  --tabs-tab-background: var(--salt-navigable-primary-background);
+  --tabs-tab-background-hover: rgba(0, 0, 0);
+  --tabs-tab-backgroundOpacity-hover: 0.08;
```

Comment:
are we waiting for tokens from design?

Reply:
the whole opacity scale is awaiting restructure, although not sure when this change will be made cc @pseys

Reply:
not yet since the 'restructure' (similar to spacing work) is on hold for now, but Paul is getting round to it soon cc @pseys :)